docs/products/kafka/howto/enable-governance.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Enable governance for Aiven for Apache Kafka®
+sidebar_label: Enable governance
+limited: true
+---
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+Enable governance in Aiven for Apache Kafka® to establish a secure and compliant framework to manage your Aiven for Apache Kafka clusters efficiently.
+
+## Prerequisites
+
+- This is a [limited availability feature](/docs/platform/concepts/beta_services). To try
+  it out, contact the sales team at [sales@aiven.io](mailto:sales@aiven.io).
```

Comment:
```suggestion
  it out, contact your  account team.
```
???

Reply:
I will leave at Sales I until I get the confirmation here.